packages/blocks/src/api/test/parser.js

```diff
@@ -137,7 +137,7 @@ describe( 'block parser', () => {
 	} );
 
 	describe( 'parseWithAttributeSchema', () => {
-		it( 'should return the matcher\'s attribute value', () => {
+		it( "should return the matcher's attribute value", () => {
```

Comment:
I would put `’` here, too. To match with other test names :)

Reply:
Agreed. I didn't want to go too wild with tests but I suppose just being consistent everywhere is cool. Will do!